lang-portal-next/frontend-next/src/components/Breadcrumbs.tsx

```
@@ -0,0 +1,80 @@
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbPage,
+  BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb"
+import { Separator } from "@/components/ui/separator"
+import { SidebarTrigger } from "@/components/ui/sidebar"
+import { useNavigation } from '@/context/NavigationContext'
+
+interface BreadcrumbItem {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'BreadcrumbItem'. Consider to delete it or rename it.

'BreadcrumbItem' is defined here:


(lint/suspicious/noRedeclare)